internal/http/handlers.go

```diff
@@ -49,6 +49,32 @@ func MetaHandler(cfg *config.Config) http.HandlerFunc {
 	}
 }
 
+func FeedHandler(config *config.Config, database db.DB, assets Assets) http.HandlerFunc {
```

Comment:
Sorry if i missed the reasoning why, but can we omit this from the current PR?

Reply:
We can, was an accidental include when separating